eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -53,10 +53,12 @@ import scala.concurrent._
   * @param datadir  directory where eclair-core will write/read its data
   * @param overrideDefaults
   * @param seed_opt optional seed, if set eclair will use it instead of generating one and won't create a seed.dat file.
+  * @param overrideElectrumServers_opt optional electrum server which, if set, supersedes the default list of electrum server.
   */
 class Setup(datadir: File,
             overrideDefaults: Config = ConfigFactory.empty(),
-            seed_opt: Option[BinaryData] = None)(implicit system: ActorSystem) extends Logging {
+            seed_opt: Option[BinaryData] = None,
+            overrideElectrumServers_opt: Option[InetSocketAddress] = None)(implicit system: ActorSystem) extends Logging {
```

Comment:
I would rather have added an `electrum` entry in `reference.conf`, and use the existing `overrideDefaults` argument to handle this.

Like this:

````
  bitcoind {
    host = "localhost"
    rpcport = 18332
    rpcuser = "foo"
    rpcpassword = "bar"
    zmq = "tcp://127.0.0.1:29000"
  }

#  electrum {
#    custom-server { // use a custom electrum server
#      host = "1.2.3.4"
#      port = 50001
#    }
#  }

  default-feerates { // those are in satoshis per kilobyte
    delay-blocks {...

```